File: Introduction_to_Programming/ITP1_1_B/Main.cs
using System;

namespace ITP
{
    class ITP_1_1_B
    {
        static void Main(string[] args)
        {
            while (true) {
                string line = Console.ReadLine();

                if (line == null) {
                    break;
                }

                int num = int.Parse(line);
                Console.WriteLine((int)Math.Pow(num, 3));
            }
        }
    }
}

File: Introduction_to_Programming/ITP1_1_B/Main.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

class Main {
    public static void main(String args[]) {
        try {
            BufferedReader stdReader =
                new BufferedReader(new InputStreamReader(System.in));
            String line;
            while ((line = stdReader.readLine()) != null) {
                int num = Integer.parseInt(line);
                System.out.println((int)Math.pow(num, 3));
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}
